runtime/mrc/src/lib.rs

```diff
@@ -300,14 +301,33 @@ impl frame_system::Config for Runtime {
 	type SS58Prefix = SS58Prefix;
 	/// The action to take on a Runtime Upgrade
 	type OnSetCode = cumulus_pallet_parachain_system::ParachainSetCode<Self>;
-	type MaxConsumers = frame_support::traits::ConstU32<16>;
+	type MaxConsumers = ConstU32<16>;
 }
 
 impl pallet_msa::Config for Runtime {
 	type Event = Event;
 	type WeightInfo = pallet_msa::weights::SubstrateWeight<Runtime>;
 }
 
+pub use common_primitives::schema::SchemaId;
+
+parameter_types! {
+	pub const MaxSchemaRegistrations: SchemaId = 1024;
+}
+
+impl pallet_schemas::Config for Runtime {
+	type Event = Event;
+	type WeightInfo = pallet_schemas::weights::SubstrateWeight<Runtime>;
+
+	// TODO: may need to be moved into MSA and import traits into schemas pallet.
+	type Signature = MultiSignature;
+	type Public = <MultiSignature as Verify>::Signer;
+
+	type MinSchemaSize = ConstU32<5>;
+	type MaxSchemaSize = ConstU32<300>;
```

Comment:
300 bytes seems low? isn't it?

Reply:
Yes, it was 255 in experiments; I'm not sure what to put there and I feel like it should be discussed since we were talking about what limits.  It's just a place holder, so I'll put a TODO there and file an issue.